CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ add_library(nlohmann_json_schema_validator
     src/json-schema-draft7.json.cpp
     src/json-uri.cpp
     src/json-validator.cpp
+    src/json_patch.cpp
```

Comment:
Please use - for filename conjunctions.

Reply:
sorry. I tried to apply your conventions. I was not focused enough and saw an underscore.